test/twine_test.coffee

```diff
@@ -277,10 +277,10 @@ suite "Twine", ->
       node = setupView(testView, context = fn: @spy())
       data = {test: 'bla123'}
 
-      $(node).trigger 'submit', data
```

Comment:
Apparently, jQuery only really triggers listeners (when using `$.trigger`) on events that jQuery has itself bound...  which really sucks to anyone trying to trigger `addEventListener` events via `$.trigger`

This test is a bit weird in that seeing `bind-event-submit="fn(data)"`implies to me (at first glance) that `data` should be existing in the current Twine context, but the previous jQuery implementation was passing `data` through with the event.  So it seems more like a test of `$.trigger(eventname, data_pojo)`.

The rest of the event handler implementation inside Twine - `wrapFunctionString(definition, '$context,$root,event,data', node).call(node, context, rootContext, event, data)` - makes the presence of the 2nd data param into a keyword of sorts that any event handler can access, I guess officially authorizing firing Twine events thru `$(node).trigger("eventname", some_pojo)`

I did a search in our erb files for `(data` and there's a few binds with this signature... and more than a few calls from coffee files where we do `$(node).trigger` with a 2nd data argument.  Perhaps an argument can be made to remove the allowance of this behaviour, but it's unintuitive to any consumer and I would suggest a `1.0.0` release.  

Maybe this is an argument to add a `Twine.trigger`?  `Twine.trigger(node, "event_name", data_pojo)` or something which dispatches the `CustomEvent` as below in the tests